tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha1AuthenticatorTests.cs

```diff
@@ -212,10 +210,6 @@ public void Constructor_should_throw_an_ArgumentNullException_when_credential_is
             sentMessages.Count.Should().Be(expectedSentMessageCount);
 
             var actualRequestIds = sentMessages.Select(m => m["requestId"].AsInt32).ToList();
-            for (var i = 0; i != actualRequestIds.Count; ++i)
-            {
-                actualRequestIds[i].Should().BeInRange(expectedRequestId + i, expectedRequestId + 10 + i);
-            }
```

Comment:
This change isn't relate to THIS ticket is it?

We could just include it... or it could be a separate ticket.

Reply:
not related. But it's helpful for the testing to reduce red tests. I would say it's ok to have this small change it in the scope of this ticket if no objections

Reply:
OK